File: mvc_comfi/mvc_comfi/Models/comfi-furnitures/Feedback.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace mvc_comfi.Models.comfi
{
    public partial class Feedback
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Suggestions { get; set; }
    }
}

File: Project01/src/services/feedback.ts
import { EmailValidator } from "@angular/forms";

export class feedback
{
  [x: string]: any;

  FirstName: string;
  LastName: string;
  Email: string;
  Suggestions: string;

  constructor(FirstName:string, LastName:string, Email:string, Suggestions:string){

    this.FirstName = FirstName;
    this.LastName = LastName;
    this.Email = Email;
    this.Suggestions = Suggestions;
}
}
